### DISCUSSION TITLE
Neo4j DB dump restore in arcade takes very long +20 hrs with missing edges.

### DISCUSSION THREAD

**Discussion by dianaoa22** [ArcadeData/arcadedb#1566] [Q&A]
The DB exported from Neo4j with 124,8M records, 124,8M relations takes considerably long time to import +20 hrs and has created only the vertices and not the edges. 

Parsed 34,653,399 (0/sec 0%) 0 records (0/sec) 34,653,375 vertices (0/sec) 0 edges (0/sec 0 skipped) 0 linked edges (0/sec 0%) updated documents 0 (0%)

+---------------+-------------------------------+
|NAME           |VALUE                          |
+---------------+-------------------------------+
|operation      |import database                |
|fromUrl        |[file:///home/arcadedb/all.jsonl|](file:///home/arcadedb/all.jsonl%7C)
|parsedRecords  |34653399                       |
|errors         |27                             |
|createdVertices|34653375                       |
|result         |OK                             |
+---------------+-------------------------------+
Command executed in 73810923ms

**Comment by lvca** [comment]
parsing 124M of vertices should be super fast usually edges are the problem with speed. Is there any chance I can have this file and test it locally?

**Reply by dianaoa22** [reply]
Thanks for your response, unfortunately I won't be able to share this dump, will try to reproduce with a generic dataset which can be shared.
